packages/web-components/src/components/content-item-horizontal/content-item-horizontal-media.ts

```diff
@@ -18,30 +20,86 @@ import { MEDIA_ALIGN } from './defs';
 const { prefix } = settings;
 const { stablePrefix: ddsPrefix } = ddsSettings;
 
+// The "lg" Carbon breakpoint.
+// @TODO See if this can be pulled from a Carbon package.
+const breakpoint: number = 1056;
```

Comment:
Is there a package I can pull this number from?

Reply:
you can see how the breakpoints are being included here: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/02ab728998f8e898300d9f15dc17e5b703a093eb/packages/web-components/src/components/card-group/card-group.ts#L15